### DISCUSSION TITLE
How to start the Workflow Diagram glsp-vscode-integration example

### DISCUSSION THREAD

**Discussion by zrwsmd** [eclipse-glsp/glsp#1081] [Q&A] [answered]
when i execute yarn install,and press F5 to launch extension,an error occired,how do i solve it,thanks a lot
![image](https://github.com/eclipse-glsp/glsp/assets/26995040/2c11953d-ea81-4f52-b827-12893c316199)

**Comment by planger** [comment]
Hi,

The error message indicates that there is no server running on the specified port, however, the lines above the error in the console indicate that the server indeed is running.

Which launch configuration did you start? I guess it was "Workflow GLSP Example Extension (Integrated Node GLSP Server)", right? The dynamically assigned port number `51672` indicates that you are using the launch config with integrated node GLSP server.

I re-tested this launch config on current master with a fresh build (`git clean -xdf && yarn install`) and this launch config worked without issues. Could there maybe be some security settings on your machine that prevent the client to connect to the GLSP server on the assigned port?

**Comment by zrwsmd** [comment]
all right,i have solve it ,thanks a lot




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Philip ***@***.***&gt;; 
发送时间: 2023年8月10日(星期四) 下午5:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [eclipse-glsp/glsp] How to start the Workflow Diagram glsp-vscode-integration example (Discussion #1081)





 
Hi,
 
The error message indicates that there is no server running on the specified port, however, the lines above the error in the console indicate that the server indeed is running.
 
Which launch configuration did you start? I guess it was "Workflow GLSP Example Extension (Integrated Node GLSP Server)", right? The dynamically assigned port number 51672 indicates that you are using the launch config with integrated node GLSP server.
 
I re-tested this launch config on current master with a fresh build (git clean -xdf &amp;&amp; yarn install) and this launch config worked without issues. Could there maybe be some security settings on your machine that prevent the client to connect to the GLSP server on the assigned port?

**Reply by rsoika** [reply]
@zrwsmd can you please describe how you solved it or what the reason was for your issue. This may help others reading this thread.

**Comment by zrwsmd** [comment, marked as answer]
&nbsp;External GLSP Server from (https://github.com/eclipse-glsp/glsp-server)and use&nbsp;&nbsp;command &nbsp;java -jar org.eclipse.glsp.example.workflow-X.X.X-SNAPSHOT-glsp.jar to launch it,then just launch Workflow GLSP Example Extension (External GLSP Server),it works well




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Ralph ***@***.***&gt;; 
发送时间: 2023年8月11日(星期五) 下午2:32
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [eclipse-glsp/glsp] How to start the Workflow Diagram glsp-vscode-integration example (Discussion #1081)





 
@zrwsmd can you please describe how you solved it or what the reason was for your issue. This may help others reading this thread.
